### PR TITLE
Added doPast in korean.js file

### PR DESCRIPTION
**Pull request by thisisrai**
Not sure if this is correct, but here is my attempt at it.

### CODE REVIEW

File: src/korean.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ class Korean {
 
   doPast(word) {
     let presentWord = doPresent(word);
-    // stuff for past tense
+    return presentWord.concat('ㅆ어')
```
Comment (cecilebertin) [inline src/korean.js]: Hi @thisisrai, thank you for contributing!

Unfortunately, I don't think you're code will work.
You have to use the breakdown and combineSymbols functions to to combine 2 or 3 jamo into one character. Here is some more information : 
http://www.programminginkorean.com/programming/hangul-in-unicode/composing-syllables-in-unicode/
Could you also add some testcases on the korean_tests.js file for the doPast function? We take the habit to write our own tests with each function we write.
This way you can then test your code by running npm run test in your terminal.

Comment (yjlim5) [reply src/korean.js]: Right, the basic steps that @cecilebertin mentioned are outlined in the issue #14 . Thanks!

Comment (cecilebertin) [reply src/korean.js]: @thisisrai, 
For next time: **Before** pushing your branch, make sure to fetch and merge the latest code and rebase your branch.

**Comment by minsooshin** [comment]
@thisisrai I already have been working on this with Heggy. I made a pull request on my own repo to get code review from her. If you don't mind, please also take a look at it. https://github.com/minsooshin/conjugator/pull/3/files
BTW, since I made this change last Sunday after the meetup, I have huge conflict with original master, but I will definitely fix it. But I would like to let Heggy take a look at it first since she has been working on this subject more a week. Is that fine for you, @thisisrai ?

**Comment by minsooshin** [comment]
@thisisrai never mind my previous comment. Since this is open source project, anyone can do whatever they want to contribute. You can go ahead with your solution. Sorry about my comment above.

**Comment by thisisrai** [comment]
Hey Mason,

No worries, I just didn't have time to review your code this week yet. I'll
definitely try to read a little bit of it prior to the meet up tomorrow.
Thank you for showing me your pull request!

Warm Regards,

Rai Lee

**Pull request closed without merging**